src/components/controls/gamepad-controls.js

```diff
@@ -0,0 +1,110 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
Though for my case, it's nice that I am able to override either the position or rotation pieces of the gamepad controls separately.

```
<a-entity controls="position: your-gamepad-controls; rotation: my-gamepad-controls">
```